src/libraries/System.Text.RegularExpressions/tools/GenerateRegexCasingTable.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
       <OutputType>Exe</OutputType>
       <TargetFramework>net7.0</TargetFramework>
```

Comment:
```suggestion
```

Reply:
@ViktorHofer you told me to revert this change https://github.com/dotnet/runtime/pull/75806#discussion_r975521050 what changed?

Reply:
Oh right, my bad.